cmc-domain/src/test/java/uk/gov/hmcts/cmc/claimstore/utils/PartyUtilsTest.java

```diff
@@ -17,22 +18,20 @@
 
 import java.util.Optional;
 
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class PartyUtilsTest {
 
     @Test
     public void getTypeMapsPartyTypesCorrectly() {
-        assertThat(PartyUtils.getType(SampleParty.builder().individual()))
+        Assertions.assertThat(PartyUtils.getType(SampleParty.builder().individual()))
```

Comment:
Use a static import please -- here and in all other places.

Reply:
Done